atd-vze/src/views/Crashes/Crashes.js

```diff
@@ -1,72 +1,166 @@
 import React from "react";
 import { withApollo } from "react-apollo";
-import TableWithFilters from "../../Components/TableWithFilters";
-import { crashDataMap } from "./crashDataMap";
 
-// Key for data response array from DB
-const dataKey = "atd_txdot_crashes";
+import GridTable from "../../Components/GridTable";
 
-const GET_CRASHES = `
-  {
-    atd_txdot_crashes(
-      offset: 0
-      where: { crash_fatal_fl: { _eq: "Y" } }
-    ) {
-      crash_id
-      death_cnt
-      tot_injry_cnt
-      crash_fatal_fl
-      rpt_street_pfx
-      rpt_street_sfx
-      rpt_street_name
-      crash_date
-    }
-  }
-`;
+import gqlAbstract from "../../queries/gqlAbstract";
 
-const FILTER_CRASHES = `
-  {
-    atd_txdot_crashes(
-      OFFSET
-      LIMIT
-      ORDER_BY
-      SEARCH
-    ) {
-      crash_id
-      death_cnt
-      tot_injry_cnt
-      crash_fatal_fl
-      rpt_street_pfx
-      rpt_street_sfx
-      rpt_street_name
-      crash_date
-    }
-  }
-`;
+// Our initial query configuration
+let queryConf = {
+  table: "atd_txdot_crashes",
+  single_item: "crashes",
+  columns: {
+    crash_id: {
+      primary_key: true,
+      searchable: true,
+      sortable: true,
+      label_search: "Search by Crash ID",
+      label_table: "Crash ID",
+      type: "Int",
+    },
+    case_id: {
+      searchable: true,
+      sortable: true,
+      label_search: "Search by Case Number",
+      label_table: "Case Number",
+      type: "String",
+    },
+    crash_date: {
+      searchable: false,
+      sortable: true,
+      label_table: "Crash Date",
+      type: "Date",
+    },
+    address_confirmed_primary: {
+      searchable: true,
+      sortable: true,
+      label_search: "Search by Primary Address",
+      label_table: "Primary Address",
+      type: "String",
+    },
+    address_confirmed_secondary: {
+      searchable: true,
+      sortable: true,
+      label_search: "Search by Secondary Address",
+      label_table: "Secondary Address",
+      type: "String",
+    },
+    tot_injry_cnt: {
+      searchable: false,
+      sortable: true,
+      label_table: "Injury Count",
+      type: "Int",
+    },
+    death_cnt: {
```

Comment:
note here that we'll need clarification on whether to display this field or the `apd_confirmed_death_count` once that is implemented. Can you make a note to get clarification?

Reply:
or just add a task to that issue? 
https://github.com/cityofaustin/atd-vz-data/issues/236